### DISCUSSION TITLE
classes/class_scenetreetimer

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#70] [User-contributed notes]
# classes/class_scenetreetimer

Inherits: RefCounted< Object One-shot timer. Description: A one-shot timer managed by the scene tree, which emits timeout on completion. See also SceneTree.create_timer. As opposed to Timer, it doe...

https://docs.godotengine.org/en/stable/classes/class_scenetreetimer.html

**Comment by Dr-Bamboo-Dev** [comment]
SceneTreeTimer objects always leak when I close the game. Is there any way to find and free them manually?

**Reply by Calinou** [reply]
This is a known issue:

- https://github.com/godotengine/godot/issues/79669
